src/workbooks/optimization/workbook.json

```diff
@@ -1,6161 +1,6618 @@
-          },
-          {
-            "id": "96d6402c-ff6c-4cf8-961d-8fbe5c98be60",
-            "cellValue": "SelectedTab",
-            "linkTarget": "parameter",
-            "linkLabel": "Compute",
-            "subTarget": "Compute",
-            "preText": "VirtualMachines",
-            "style": "link"
-          },
-          {
-            "id": "34014531-76a5-4eab-9772-9944fa241fea",
-            "cellValue": "SelectedTab",
-            "linkTarget": "parameter",
-            "linkLabel": "Azure Hybrid Benefit",
-            "subTarget": "AHUB",
-            "style": "link"
-          },
-          {
-            "id": "4f4786a7-c2df-4788-b5b6-63057bc0c24a",
-            "cellValue": "SelectedTab",
-            "linkTarget": "parameter",
-            "linkLabel": "Storage",
-            "subTarget": "Storage",
-            "style": "link"
-          },
-          {
-            "id": "69dfc3f7-5937-49ba-9851-d6d2973a0c69",
-            "cellValue": "SelectedTab",
-            "linkTarget": "parameter",
-            "linkLabel": "Networking",
-            "subTarget": "Networking",
-            "style": "link"
-          }
-        ]
-      },
-      "name": "links - MainTabs"
-    },
-    {
-      "type": 9,
-      "content": {
-        "version": "KqlParameterItem/1.0",
-        "crossComponentResources": [
-          "{Subscription}"
-        ],
-        "parameters": [
-          {
-            "id": "51aa3a9b-14e0-4c22-a60d-abdbf8813f00",
-            "version": "KqlParameterItem/1.0",
-            "name": "Subscription",
-            "type": 6,
-            "isRequired": true,
-            "multiSelect": true,
-            "quote": "'",
-            "delimiter": ",",
-            "typeSettings": {
-              "additionalResourceOptions": [
-                "value::all"
-              ],
-              "includeAll": true,
-              "showDefault": false
-            },
-            "timeContext": {
-              "durationMs": 86400000
-            },
-            "defaultValue": "value::all",
-            "label": " Subscription"
-          },
-          {
-            "id": "f342a111-002a-47fd-807f-0d4ccac0618a",
-            "version": "KqlParameterItem/1.0",
-            "name": "ResourceGroup",
-            "type": 2,
-            "multiSelect": true,
-            "quote": "'",
-            "delimiter": ",",
-            "query": "resourceContainers | where type == 'microsoft.resources/subscriptions/resourcegroups'\r\n| project resourceGroup\r\n| sort by resourceGroup asc",
-            "crossComponentResources": [
-              "{Subscription}"
-            ],
-            "typeSettings": {
-              "additionalResourceOptions": [
-                "value::all"
-              ],
-              "showDefault": false
-            },
-            "defaultValue": "value::all",
-            "queryType": 1,
-            "resourceType": "microsoft.resourcegraph/resources",
-            "label": "Resource group"
```

Comment:
Revert these casing changes. They don't align to the Microsoft style guide.

Reply:
so those labels were added by the UX team, we would need to align so that we don't have to change all the workbook labels every time there is a new version. Is "Resource Group" in this case acceptable ?

Reply:
No, "Resource Group" is not acceptable. Microsoft's style guide dictates that we use sentence casing. See https://learn.microsoft.com/style-guide/capitalization